lib/bloc/fee_options/fee_options_bloc.dart

```diff
@@ -21,7 +21,7 @@ class FeeOptionsBloc extends Cubit<FeeOptionsState> {
     try {
       _log.info("Estimate reverse swap fees for: $sendAmountSat");
       final req = ReverseSwapFeesRequest(sendAmountSat: sendAmountSat);
-      ReverseSwapPairInfo reverseSwapPairInfo = await _breezLib.fetchReverseSwapFees(req: req);
+      ReverseSwapPairInfo reverseSwapPairInfo = await _breezLib.fetchReverseSwapFees(req);
```

Comment:
Please revert this, I'll make fetchReverseSwapFees to use req as named parameter

Reply:
Done